src/main/java/Main.java

```diff
@@ -1,22 +1,95 @@
-import java.io.File;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
 import java.io.FileWriter;
 import java.io.IOException;
 
 public class Main {
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) {
+        final String employeesFile = "employees.csv";
+        final String statsEmployeesFile = "employees.csv";
 
-        File file = new File("employees.csv");
+        Employee[] employees = new Employee[0];
+        try {
+            employees = EmployeesInfoReader.readFile("employees.csv");
+        } catch (FileNotFoundException e) {
+            System.err.println("Brak pliku o nazwie: " + employeesFile);
+        }
+
+        double avgSalary = getAverageSalary(employees);
+        System.out.println("Średnia wypłata: " + avgSalary);
+        double lowestSalary = getLowestSalary(employees);
+        System.out.println("Minimalna wypłata: " + lowestSalary);
+        double highestSalary = getHighestSalary(employees);
+        System.out.println("Maksymalna wypłata: " + highestSalary);
+        int employeesDepartment = getEmployeesDepartment(employees, "IT");
+
+        Stats stats = new Stats(avgSalary, lowestSalary, highestSalary, employeesDepartment);
+
+        try {
+            saveToFile("stats.txt", stats);
+        } catch (IOException e) {
+            System.err.println("Nie udało się dodać danych do pliku o nazwie: " + statsEmployeesFile);
+        }
+    }
 
-        if (file.exists()) {
-            File stats = new File("stats.txt");
-            stats.createNewFile();
+    private static void saveToFile(String statsEmployeesFile, Stats stats) throws IOException {
+        BufferedWriter writer = new BufferedWriter(new FileWriter(statsEmployeesFile));
+        writer.write((int) stats.getAvgSalary());
+        writer.newLine();
+        writer.write((int) stats.getLowestSalary());
+        writer.newLine();
+        writer.write((int) stats.getHighestSalary());
+        writer.newLine();
+        writer.write(stats.getEmployeesDepartment());
+        writer.newLine();
+        writer.close();
+    }
 
-//            FileWriter fileWriter = new FileWriter(stats);
-//            fileWriter.write("Średnia wypłata: 5000");
-//
-//            fileWriter.close();
+    private static int getEmployeesDepartment(Employee[] employees, String department) {
+        int it = 0;
+        int support = 0;
+        int management = 0;
+        for (Employee employee : employees) {
+            if (employee.getDepartment().equals("IT")) {
+                it++;
+            } else if (employee.getDepartment().equals("Support")) {
+                support++;
+            } else if (employee.getDepartment().equals("Management")) {
+                management++;
+            }
         }
+        return it + support + management;
+    }
```

Comment:
Ta metoda nie powinna liczyć sumy pracownikach we wszystkich działach (bo wtedy to będzie to samo co `employees.length`). 
Powinieneś mieć takiego ifa:
```
if (employee.getDepartment().equals(department))
```
i później z tej metody korzystać np. tak:
```
int itEmployees = getEmployeesDepartment(employees, "IT");
int supportEmployees = ...
```
i w klasie Stats pwinny być 3 osobne pola z liczbą pracowników w poszczególnych działach